concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -266,13 +286,13 @@ def test_case_2b_not_enough_funds(cluster_consts, cluster_url, test_id):
         force_subtask_results(
             timestamp=timestamp_to_isoformat(current_time),
             ack_report_computed_task=ack_report_computed_task(
-                timestamp=timestamp_to_isoformat(current_time - (SUBTASK_VERIFICATION_TIME * 1.4)),
+                timestamp=timestamp_to_isoformat(current_time),
                 report_computed_task=report_computed_task(
                     task_to_compute=create_signed_task_to_compute(
-                        timestamp=timestamp_to_isoformat(current_time - SUBTASK_VERIFICATION_TIME * 1.5),
+                        timestamp=calculate_timestamp(current_time, cluster_consts),
                         task_id=task_id,
                         subtask_id=subtask_id,
-                        deadline=current_time - (SUBTASK_VERIFICATION_TIME),
+                        deadline=current_time - (1 + ((4 * cluster_consts.concent_messaging_time) + 183)),  # svt 40183
```

Comment:
Same here - please try to express this via new `calculate_deadline`

Reply:
:+1: done